src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs

```diff
@@ -541,6 +541,44 @@ public async Task<NotificationHubJob> GetNotificationHubJobAsync(string jobId, s
             };
         }
 
+        /// <summary>Gets the notification hub jobs asynchronously.</summary>
+        /// <param name="notificationHubPath">The notification hub path.</param>
+        /// <returns>A task that represents the asynchronous get jobs operation</returns>
+        public async Task<IEnumerable<NotificationHubJob>> GetNotificationHubJobsAsync(string notificationHubPath)
+        {
+            var requestUri = new UriBuilder(Address)
+            {
+                Scheme = Uri.UriSchemeHttps,
+                Path = $"{notificationHubPath}/jobs",
+                Query = $"?api-version={ApiVersion}"
+            };
+
+            using (var response = await SendAsync(() => CreateHttpRequest(HttpMethod.Get, requestUri.Uri)).ConfigureAwait(false))
+            {
+                var result = new List<NotificationHubJob>();
```

Comment:
Could you please move this to private function. We have exactly the same code in GetNotificationHubsAsync method